Classes/Console/Database/Schema/SchemaUpdateType.php

```diff
@@ -14,14 +14,24 @@
  *
  */
 
-use TYPO3\CMS\Core\Type\Enumeration;
+use TYPO3\CMS\Core\Type\Exception\InvalidEnumerationDefinitionException;
 use TYPO3\CMS\Core\Type\Exception\InvalidEnumerationValueException;
 
 /**
  * List of database schema update types
```

Comment:
Please add a `@todo` here that this should be converted into native enums. For now keeping the current behavior 1:1 is fine however.